test/fixtures/doc-yaml/user-guide/update-demo/kitten-rc.yaml

```diff
@@ -13,7 +13,7 @@ spec:
         version: kitten
     spec:
       containers:
-      - image: gcr.io/google_containers/update-demo:kitten
+      - image: {{kittenImage}}
```

Comment:
Is this file actually used by e2e tests?
If it is a template, the filename must indicate that (may it be with `.in` or whatever)
If it's used by the e2e tests and intended to be a template, please use the normal `{{ .Field }}` syntax